### ISSUE TITLE
[Req] Addittional message that indicates in which language section contributor is in (Speak/Write)

### ISSUE THREAD

**Issue by robovoice1** [common-voice/common-voice#4167]
**Is your feature request related to a problem? Please describe.**

It appears to be not clear for some contributors in which language section they are  contributing (clips and sentences).

A) For more than half a year there are many not english recorded clips in the english listen section
B) Many non english sentences in english review section (up to 50 per day!)
![Screenshot_20230910-142225](https://github.com/common-voice/common-voice/assets/92784377/5e38502b-5a7c-46d9-94c3-a6c1ad6ca9fe)

**Describe the solution you'd like**

A green or red pop up message in Speak and Write is displayed constantly (as long as the contributor is in Speak and Write secion) for every language (and in non english sections a translated version of this pop up message) which clearly indicates the contributor in what language section he is contributing the clips and sentences.

Please record and submit this sentence in english.(Speak)
Please write and submit sentence(s)  in english language only. (Write)

**Describe alternatives you've considered**

Leaving this as is and those contributors  will never change their behavior

**Additional context**

Normally this should be clear with the selected ui, but some contributors do not know how to change it ???

**Comment by jessicarose** [comment]
Thanks so much for taking the time to share your idea for a feature request to improve visibility of which language users are in while they're making contributions. I'll share this with our front end team to help better inform their future roadmap around UI improvements and user experience.

**Comment by robovoice1** [comment]
It is good to read, that this proposal is now "on the radar" of moz/cv 😃

**Comment by moz-kathyreid** [comment]
I agree the current behaviour is confusing. I have two languages in my Profile - `en` and `sw` - and here I am given a `sw` sentence to validate, but `en` is showing in the top right hand corner. 

![Image](https://github.com/user-attachments/assets/f6769cae-95e5-4133-95c7-b083704f8bf1)
